docs/reference/modules/piece-page-type.md

```diff
@@ -0,0 +1,240 @@
+---
+extends: '@apostrophecms/doc-type'
+---
+
+# `@apostrophecms/piece-page-type`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module cooperates with the 'piece-type' module to expose two specialized views. The [index page](https://v3.docs.apostrophecms.org/reference/glossary.html#index-page) displays all pieces of a particular `piece-type` in a paginated, filterable manner. The [show page](https://v3.docs.apostrophecms.org/reference/glossary.html#show-page) is for presenting individual pieces. These features are added to those exposed by the  ['@apostrophecms/page'](/reference/modules/page.md) module.
```

Comment:
Once an editor adds a page of this type to the site via user interface, it becomes possible to view a listing of pieces by visiting that page's URL, and to view individual pieces of the relevant type by adding the slug of any piece to the page's URL, like this: `/slug/of/page/slug-of-piece`

This default behavior can be customized as described below.

It is possible to add more than one such page to the site for the same type, and to add custom logic to decide which pieces should be associated with each such page.

Reply:
Added.